templates/widget/widget.test.ts

```diff
@@ -0,0 +1,36 @@
+import 'reflect-metadata';
+import { MessageService } from '@theia/core';
+import { ContainerModule, Container } from 'inversify';
+import { <%= params.extensionPrefix %>Widget } from './<%= params.extensionPath %>-widget';
+import { render } from '@testing-library/react'
+
+describe('<%= params.extensionPrefix %> widget extension unit tests', () => {
```

Comment:
I believe it should only be the name of the widget?

```suggestion
describe('<%= params.extensionPrefix %>Widget', () => {
```